src/js/features/clear-clutter.js

```diff
@@ -13,4 +13,9 @@ module.exports = function() {
         removeElement('#nav_related_streams');
         $('body').append('<style>#nav_games, #nav_streams, #nav_related_streams, .js-recommended-channels { display: none !important; }</style>');
     }
+
+    // if setting "Hide Last Broadcast Banner" is true
+    if (bttv.settings.get('hideLastBroadcast') === true) {
```

Comment:
Suggestions for better location? 

And for the CSS in js, I mainly did it because I saw it elsewhere and also to modify the minimal amount of files.